### ISSUE TITLE
could not open 'C:\Users\FoxMaccloud\source\vcpkg\scripts\buildsystems\msbuild\..\..\..\installed\x64-windows\lib\*.lib': invalid argument	SkyrimMod	C:\Users\FoxMaccloud\source\repos\SkyrimMod\SkyrimMod\lld-link	1

### ISSUE THREAD

**Issue by FoxMaccloud** [Caphyon/clang-power-tools#950]
I get an error when trying to compile using LLVM.
For some reason it works fine for x86, but not for x64.

I first tired to install LLVM via vcpkg and that's when I first got this error:
```
1>lld-link : error : could not open 'C:\Users\FoxMaccloud\source\vcpkg\scripts\buildsystems\msbuild\..\..\..\installed\x64-windows\lib\*.lib': invalid argument
1>Done building project "SkyrimMod.vcxproj" -- FAILED.
```

I then tried to install the Clang Power Tools in hopes that this solved the issue.
I have no idea what's wrong. The error I get from Power Tools is very similar to the one I get from vcpkg.
```
could not open 'C:\Users\FoxMaccloud\source\vcpkg\scripts\buildsystems\msbuild\..\..\..\installed\x64-windows\lib\*.lib': invalid argument	SkyrimMod	C:\Users\FoxMaccloud\source\repos\SkyrimMod\SkyrimMod\lld-link	1
```
Not sure if it's a bug or I'm doing anything wrong.

Best regards,
FoxMaccloud

**Comment by hero101111** [comment]
Hi @FoxMaccloud ,

Can you provide more info on your setup? (VS version, LLVM version)
We have been unable to reproduce this issue so far.

Regards,
Gabriel.

**Comment by FoxMaccloud** [comment]
Hi.

I have since been able to solve the issue myself by getting the compiler via MSVS installer instead of getting it via VCPKG.
The steps I did to produce the issue was installing via vcpkg using this command in powershell: `.\vcpkg.exe install llvm --triplet x64-windows`

I hope this helps!

If you have any more questions that can help you out, feel free to take contact.

Best regards,
FoxMaccloud
